data/complete.queryMap.json

```diff
@@ -152,58 +156,5 @@
-  "481a8e558a410ee5a700ad3c29d552fb": "query FairTestsQuery {\n  fair(id: \"sofa-chicago-2018\") {\n    ...Fair_fair\n    id\n  }\n}\n\nfragment Fair_fair on Fair {\n  id\n  ...FairDetail_fair\n}\n\nfragment FairDetail_fair on Fair {\n  ...FairHeader_fair\n  slug\n  internalID\n  name\n  hours\n  isActive\n  location {\n    ...LocationMap_location\n    coordinates {\n      lat\n      lng\n    }\n    id\n  }\n  organizer {\n    website\n    id\n  }\n  profile {\n    name\n    id\n  }\n  sponsoredContent {\n    pressReleaseUrl\n    activationText\n  }\n  shows: showsConnection(first: 5) {\n    pageInfo {\n      hasNextPage\n      startCursor\n      endCursor\n    }\n    edges {\n      cursor\n      node {\n        artworks: artworksConnection(first: 4) {\n          edges {\n            node {\n              slug\n              id\n            }\n          }\n        }\n        ...FairBoothPreview_show\n        id\n        __typename\n      }\n    }\n  }\n}\n\nfragment FairHeader_fair on Fair {\n  slug\n  name\n  formattedOpeningHours\n  startAt\n  endAt\n  exhibitionPeriod\n  counts {\n    artists\n  }\n  image {\n    url\n  }\n  followedContent {\n    artists {\n      name\n      href\n      slug\n      internalID\n      id\n    }\n  }\n  artistsConnection(first: 3) {\n    edges {\n      node {\n        name\n        href\n        slug\n        internalID\n        id\n      }\n    }\n  }\n  profile {\n    id\n    icon {\n      url(version: \"square140\")\n    }\n  }\n}\n\nfragment LocationMap_location on Location {\n  id\n  internalID\n  city\n  address\n  address_2: address2\n  postal_code: postalCode\n  summary\n  coordinates {\n    lat\n    lng\n  }\n}\n\nfragment FairBoothPreview_show on Show {\n  slug\n  internalID\n  counts {\n    artworks\n  }\n  partner {\n    __typename\n    ... on Partner {\n      name\n      href\n      slug\n      internalID\n      id\n      profile {\n        id\n        slug\n        internalID\n        isFollowed\n      }\n    }\n    ... on Node {\n      id\n    }\n    ... on ExternalPartner {\n      id\n    }\n  }\n  coverImage {\n    url\n  }\n  location {\n    display\n    id\n  }\n  artworks: artworksConnection(first: 4) {\n    edges {\n      node {\n        ...GenericGrid_artworks\n        id\n      }\n    }\n  }\n}\n\nfragment GenericGrid_artworks on Artwork {\n  id\n  image {\n    aspect_ratio: aspectRatio\n  }\n  ...ArtworkGridItem_artwork\n}\n\nfragment ArtworkGridItem_artwork on Artwork {\n  title\n  date\n  sale_message: saleMessage\n  is_biddable: isBiddable\n  is_acquireable: isAcquireable\n  is_offerable: isOfferable\n  slug\n  sale {\n    is_auction: isA
```

Comment:
Seems like quite a few deletions here, is that expected?

Reply:
Yeah. I regenerated the query map from scratch, which is safe to do. The queries removed here are no longer used in emission. The reason they were in there is because relay-compiler will treat this file as append-only, rather than deleting old unused queries. We don't need to rely on that logic because we implemented our own query-preservation logic layer that gets applied only to the metaphysics repo (when deploying a new version of emission).

If there were a relay-compiler option to overwrite the previous file then we'd want to use that in emission, but since there isn't I guess we occasionally have this kind of churn. I'll add that to my list of things to fix in the relay ecosystem ☑️